lib/agent0/agent0/hyperdrive/policies/zoo.py

```diff
@@ -1,11 +1,15 @@
-"""Policies for expert system trading bots"""
+"""Policies for expert system trading bots."""
 from __future__ import annotations
 
 from typing import NamedTuple
 
 from .arbitrage import Arbitrage
 from .random import Random
 from .smart_long import SmartLong
+from .lpandarb import LPandArb
+
+# Actual policies
```

Comment:
can you delete this "Actual policies" comment? not necessary & mixes up the imports.